rest/endpoints/user_recordings/profile_recordings.yml

```diff
@@ -776,19 +847,21 @@ paths:
                   data:
                     type: array
                     items:
-                      $ref: "https://raw.githubusercontent.com/booxmedialtd/api-documentation/master/rest/snippets/UserRecordings/RecordingGroupEntity.yml#/RecordingGroupEntity"
+                      $ref: "../../snippets/UserRecordings/RecordingGroupEntity.yml#/RecordingGroupEntity"
                   included:
                     type: array
                     items:
-                      $ref: "https://raw.githubusercontent.com/booxmedialtd/api-documentation/master/rest/snippets/EnrichedMetadata/SeriesEntity.yml#/SeriesEntity"
+                      anyOf:
+                        - $ref: "../../snippets/EnrichedMetadata/SeriesEntity.yml#/SeriesEntity"
+                        - $ref: "../../snippets/EnrichedMetadata/ChannelV4Entity.yml#/ChannelEntity"
 
   # ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 
   /user_profiles/{user_profile_id}/recordings/recording_groups/{recording_group_id}/recordings:
     get:
       description: |
         * **Available since:** 2019-11 (BPLAT-10674)
-      summary: An index of the recordings for a specific recording group
+      summary: An index of the recordings for a specific recording group. This endpoint will be deprecated soon and it is not recommended to build new systems using it.
```

Comment:
Let's say it _is_ deprecated, as it will be when we publish this new definition.